GestaoFinanceira/GestaoFinanceira/Controller/CrtSubCategories.cs

```diff
@@ -0,0 +1,31 @@
+﻿using GestaoFinanceira.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GestaoFinanceira.Controller
+{
+    class CrtSubCategories
+    {
+        public List<SubCategories> List()
+        {
+            return null;
+        }
+        public SubCategories Find(int categoriesId)
+        {
+            return null;

```

Comment:
Quando prevemos um código, porém, não o implementamos, o ideal é definir ele, entretanto às vezes, é necessário já deixá-lo definido para poder desenrolar outras features. Neste caso, é uma boa prática, ao invez de retornar nulo, gerar uma exceção. 

```suggestion
           throw new NotImplementedException();
```

Desta forma, você evita que alguém chame esse método acreditando que está implementado.

Reply:
Ahh que legal, não conhecia esse método de passar uma Exception em um método não implementado. Obrigado pelos apontamentos!